libs/vre/resource-editor/resource-properties/src/lib/value-components/link-value.component.ts

```diff
@@ -137,13 +131,14 @@ export class LinkValueComponent implements OnInit, AfterViewInit, OnDestroy {
   openCreateResourceDialog(event: any, resourceClassIri: string, resourceType: string) {
     let myResourceId: string;
     event.stopPropagation();
-    const projectIri = (this._store.selectSnapshot(ProjectsSelectors.currentProject) as ReadProject)?.id;
+    const project = this._store.selectSnapshot(ProjectsSelectors.currentProject)!;
```

Comment:
undefined if routed to the resource without prior project context, e.g. https://app.dev.dasch.swiss/resource/0838/WGp1GewbTOONchBiW6q6wg